src/phoenix/db/insertion/helpers.py

```diff
@@ -26,29 +39,57 @@ class OnConflict(Enum):
 
 
 def insert_on_conflict(
+    *records: Mapping[str, Any],
+    table: Type[Base],
     dialect: SupportedSQLDialect,
-    table: Any,
-    values: Mapping[str, Any],
-    constraint: str,
-    column_names: Sequence[str],
-    on_conflict: OnConflict = OnConflict.DO_NOTHING,
+    unique_by: Sequence[str],
```

Comment:
```suggestion
    *records: Mapping[str, Any],
    /,
    *,
    table: Type[Base],
    dialect: SupportedSQLDialect,
    unique_by: Sequence[str],
```

Reply:
i think this is equivalent and implied (otherwise the variadic positional args would not work)